build_scripts/windows/scripts/build_saw.cmd

```diff
@@ -165,12 +165,14 @@ popd
 rm %CLI_SRC%\azure-cli\requirements.py3.windows-spython.txt
 %PYTHON_EXE% -m pip install "%MISC_DIR%\%WINDOWS_HTTP_FILENAME%" --no-warn-script-location --force-reinstall --target %SPYTHON_LIB%
 
+@REM Install portalocker free version msal_extensions
+%PYTHON_EXE% -m pip install https://github.com/AzureAD/microsoft-authentication-extensions-for-python/archive/optional-portalocker.zip --no-warn-script-location --ignore-installed --target %SPYTHON_LIB%
```

Comment:
I never knew we can download a ZIP from this location. Any document for it?

Reply:
`pip` has official support for installing from git repo: https://pip.pypa.io/en/stable/topics/vcs-support/

Reply:
I guess it's documented before GitHub REST API. I'll change it to GitHub url.